<?xml version="1.0" encoding="utf-8"?>
<TestRecord xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <Tests>
    <TestRecord Name="Qiniu">
      <Tests>
        <TestRecord Name="Test">
          <Tests>
            <TestRecord Name="IOClientTest">
              <Tests>
                <TestRecord Name="PutFileTest">
                  <Results>
                    <UnitTestResult>
                      <TestDate>2013-07-29T18:43:00</TestDate>
                      <Status />
                      <Passed>1</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Test successful

Execution time: 0.01ms</Message>
                      <ConsoleOutput />
                      <ConsoleError />
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T21:31:19</TestDate>
                      <Status>Failure</Status>
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Object reference not set to an instance of an object</Message>
                      <StackTrace>  at MonoDevelop.NUnit.UnitTestResult.CreateFailure (System.String message, System.Exception ex) [0x00017] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTestResult.cs:67 
  at MonoDevelop.NUnit.NUnitAssemblyTestSuite.RunUnitTest (MonoDevelop.NUnit.UnitTest test, System.String suiteName, System.String pathName, System.String testName, MonoDevelop.NUnit.TestContext testContext) [0x00240] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitAssemblyTestSuite.cs:440 
  at MonoDevelop.NUnit.NUnitTestCase.OnRun (MonoDevelop.NUnit.TestContext testContext) [0x0001b] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitTestCase.cs:55 
  at MonoDevelop.NUnit.UnitTest.Run (MonoDevelop.NUnit.TestContext testContext) [0x00020] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTest.cs:361 </StackTrace>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T21:34:04</TestDate>
                      <Status>Failure</Status>
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Object reference not set to an instance of an object</Message>
                      <StackTrace>  at MonoDevelop.NUnit.UnitTestResult.CreateFailure (System.String message, System.Exception ex) [0x00017] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTestResult.cs:67 
  at MonoDevelop.NUnit.NUnitAssemblyTestSuite.RunUnitTest (MonoDevelop.NUnit.UnitTest test, System.String suiteName, System.String pathName, System.String testName, MonoDevelop.NUnit.TestContext testContext) [0x00240] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitAssemblyTestSuite.cs:440 
  at MonoDevelop.NUnit.NUnitTestCase.OnRun (MonoDevelop.NUnit.TestContext testContext) [0x0001b] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitTestCase.cs:55 
  at MonoDevelop.NUnit.UnitTest.Run (MonoDevelop.NUnit.TestContext testContext) [0x00020] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTest.cs:361 </StackTrace>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T21:36:43</TestDate>
                      <Status>Failure</Status>
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Object reference not set to an instance of an object</Message>
                      <StackTrace>  at MonoDevelop.NUnit.UnitTestResult.CreateFailure (System.String message, System.Exception ex) [0x00017] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTestResult.cs:67 
  at MonoDevelop.NUnit.NUnitAssemblyTestSuite.RunUnitTest (MonoDevelop.NUnit.UnitTest test, System.String suiteName, System.String pathName, System.String testName, MonoDevelop.NUnit.TestContext testContext) [0x00240] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitAssemblyTestSuite.cs:440 
  at MonoDevelop.NUnit.NUnitTestCase.OnRun (MonoDevelop.NUnit.TestContext testContext) [0x0001b] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitTestCase.cs:55 
  at MonoDevelop.NUnit.UnitTest.Run (MonoDevelop.NUnit.TestContext testContext) [0x00020] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTest.cs:361 </StackTrace>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T21:37:38</TestDate>
                      <Status />
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>1</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>  PutFileTest Failure
  Expected: True
  But was:  False
</Message>
                      <StackTrace>  at NUnit.Framework.Assert.That (System.Object actual, IResolveConstraint expression, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
  at NUnit.Framework.Assert.IsTrue (Boolean condition, System.String message) [0x00000] in &lt;filename unknown&gt;:0 
  at Qiniu.Test.IOClientTest.PutFileTest () [0x0009f] in /Users/icattlecoder/qiniu/csharp-sdk/Qiniu.Test/MyClass.cs:32 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                      <ConsoleOutput>gogopher.jpg
</ConsoleOutput>
                      <ConsoleError />
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T21:37:56</TestDate>
                      <Status>Failure</Status>
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Object reference not set to an instance of an object</Message>
                      <StackTrace>  at MonoDevelop.NUnit.UnitTestResult.CreateFailure (System.String message, System.Exception ex) [0x00017] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTestResult.cs:67 
  at MonoDevelop.NUnit.NUnitAssemblyTestSuite.RunUnitTest (MonoDevelop.NUnit.UnitTest test, System.String suiteName, System.String pathName, System.String testName, MonoDevelop.NUnit.TestContext testContext) [0x00240] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitAssemblyTestSuite.cs:440 
  at MonoDevelop.NUnit.NUnitTestCase.OnRun (MonoDevelop.NUnit.TestContext testContext) [0x0001b] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitTestCase.cs:55 
  at MonoDevelop.NUnit.UnitTest.Run (MonoDevelop.NUnit.TestContext testContext) [0x00020] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTest.cs:361 </StackTrace>
                    </UnitTestResult>
                  </Results>
                </TestRecord>
                <TestRecord Name="PutTest">
                  <Results>
                    <UnitTestResult>
                      <TestDate>2013-07-29T23:29:49</TestDate>
                      <Status />
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>1</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>  PutFileTest Failure
  Expected: True
  But was:  False
</Message>
                      <StackTrace>  at NUnit.Framework.Assert.That (System.Object actual, IResolveConstraint expression, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
  at NUnit.Framework.Assert.IsTrue (Boolean condition, System.String message) [0x00000] in &lt;filename unknown&gt;:0 
  at Qiniu.Test.IOClientTest.PutTest () [0x000a3] in /Users/icattlecoder/qiniu/csharp-sdk/Qiniu.Test/IOClientTest.cs:54 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                      <ConsoleOutput>gogopher.jpg
System.Net.WebException: The remote server returned an error: (614) status code 614.
  at System.Net.HttpWebRequest.CheckFinalStatus (System.Net.WebAsyncResult result) [0x00000] in &lt;filename unknown&gt;:0 
  at System.Net.HttpWebRequest.SetResponseData (System.Net.WebConnectionData data) [0x00000] in &lt;filename unknown&gt;:0 
</ConsoleOutput>
                      <ConsoleError />
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T23:30:43</TestDate>
                      <Status>Failure</Status>
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>0</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>Object reference not set to an instance of an object</Message>
                      <StackTrace>  at MonoDevelop.NUnit.UnitTestResult.CreateFailure (System.String message, System.Exception ex) [0x00017] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTestResult.cs:67 
  at MonoDevelop.NUnit.NUnitAssemblyTestSuite.RunUnitTest (MonoDevelop.NUnit.UnitTest test, System.String suiteName, System.String pathName, System.String testName, MonoDevelop.NUnit.TestContext testContext) [0x00240] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitAssemblyTestSuite.cs:440 
  at MonoDevelop.NUnit.NUnitTestCase.OnRun (MonoDevelop.NUnit.TestContext testContext) [0x0001b] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/NUnitTestCase.cs:55 
  at MonoDevelop.NUnit.UnitTest.Run (MonoDevelop.NUnit.TestContext testContext) [0x00020] in /Users/builder/data/lanes/monodevelop-lion-monodevelop-4.0.10-branch/f324e215/source/monodevelop/main/src/addins/NUnit/Services/UnitTest.cs:361 </StackTrace>
                    </UnitTestResult>
                    <UnitTestResult>
                      <TestDate>2013-07-29T23:34:24</TestDate>
                      <Status />
                      <Passed>0</Passed>
                      <Errors>0</Errors>
                      <Failures>1</Failures>
                      <Inconclusive>0</Inconclusive>
                      <NotRunnable>0</NotRunnable>
                      <Skipped>0</Skipped>
                      <Ignored>0</Ignored>
                      <Time />
                      <Message>  PutFileTest Failure
  Expected: True
  But was:  False
</Message>
                      <StackTrace>  at NUnit.Framework.Assert.That (System.Object actual, IResolveConstraint expression, System.String message, System.Object[] args) [0x00000] in &lt;filename unknown&gt;:0 
  at NUnit.Framework.Assert.IsTrue (Boolean condition, System.String message) [0x00000] in &lt;filename unknown&gt;:0 
  at Qiniu.Test.IOClientTest.PutTest () [0x0008c] in /Users/icattlecoder/qiniu/csharp-sdk/Qiniu.Test/IO/IOClientTest.cs:53 
  at (wrapper managed-to-native) System.Reflection.MonoMethod:InternalInvoke (System.Reflection.MonoMethod,object,object[],System.Exception&amp;)
  at System.Reflection.MonoMethod.Invoke (System.Object obj, BindingFlags invokeAttr, System.Reflection.Binder binder, System.Object[] parameters, System.Globalization.CultureInfo culture) [0x00000] in &lt;filename unknown&gt;:0 </StackTrace>
                      <ConsoleOutput>gogopher.jpg
System.Net.WebException: The remote server returned an error: (614) status code 614.
  at System.Net.HttpWebRequest.CheckFinalStatus (System.Net.WebAsyncResult result) [0x00000] in &lt;filename unknown&gt;:0 
  at System.Net.HttpWebRequest.SetResponseData (System.Net.WebConnectionData data) [0x00000] in &lt;filename unknown&gt;:0 
</ConsoleOutput>
                      <ConsoleError />
                    </UnitTestResult>
                  </Results>
                </TestRecord>
              </Tests>
            </TestRecord>
          </Tests>
        </TestRecord>
      </Tests>
    </TestRecord>
  </Tests>
</TestRecord>